variables.tf

```diff
@@ -1,4 +1,111 @@
-variable "example" {
-  description = "Example variable"
-  default     = "hello world"
+variable "account_enabled" {
+  type        = bool
+  description = "Whether to enable Amazon Macie and start all Macie activities for the account."
+  default     = true
+}
+
+variable "admin_account_ids" {
+  type        = list(string)
+  description = "The list of AWS account IDs for the account to designate as the delegated Amazon Macie administrator accounts for the organization."
+  default     = []
+}
+
+variable "members" {
+  type        = list(any)
+  default     = []
```

Comment:
nit. consistency. it should be type, description, default.

Reply:
my preference `type`, `description`, `default`